src/specs/components/mega-menu/demo/data.js

```diff
@@ -36,6 +36,7 @@ module.exports = {
       path: exampleLink,
       info: {
         title: 'About the News and Media',
+        title_id: 'about-the-news-id',
```

Comment:
do we really need to set a new parameter for this? Couldn't we just generate a unique id based on the global mega menu id?

Reply:
it's the usual situation where we need to generate ids but we cannot use those for the tests, if you prefer i can remove these from the specs and only add them in the tests, in the template if this is not provided an automatic id is generated